ckan/tests/controllers/test_home.py

```diff
@@ -56,6 +56,13 @@ def test_email_address_no_nag(self):
 
         assert 'add your email address' not in response
 
+    @helpers.change_config('ckan.legacy_pylons_routes',
+                           '{"home":"home.index", "about": "home.about"}')
```

Comment:
That test doesn't tell us much, because the values you are passing are the same ones used by default. A better test would be to set `@helpers.change_config('ckan.legacy_pylons_routes', '{"my_home_route":"home.index"}')` and see if both `url_for('home')` and `url_for('my_home_route')` still work.